pkg/terraform/install_test.go

```diff
@@ -26,34 +27,52 @@ func TestMixin_UnmarshalInstallStep(t *testing.T) {
 	b, err := ioutil.ReadFile("testdata/install-input.yaml")
 	require.NoError(t, err)
 
-	var step InstallStep
-	err = yaml.Unmarshal(b, &step)
+	var action InstallAction
+	err = yaml.Unmarshal(b, &action)
 	require.NoError(t, err)
+	require.Len(t, action.Steps, 1)
+	step := action.Steps[0]
 
 	assert.Equal(t, "Install MySQL", step.Description)
+	assert.Equal(t, true, step.Init)
+	assert.Equal(t, "TRACE", step.LogLevel)
 }
 
 func TestMixin_Install(t *testing.T) {
 	installTests := []InstallTest{
 		{
-			expectedCommand: "terraform apply --help",
-			installStep: InstallStep{},
+			expectedCommand: fmt.Sprintf(
+				"terraform apply -auto-approve -var cool=true -var foo=bar %s", DefaultWorkingDir),
+			installStep: InstallStep{
+				InstallArguments: InstallArguments{
+					AutoApprove: true,
+					Init:        false,
+					LogLevel: "TRACE",
+					Vars: map[string]string{
+						"cool": "true",
+						"foo":  "bar",
+					},
+				},
+			},
 		},
+		// TODO: test Init: true (requires main test helper refactor to support one action issuing multiple commands)
```

Comment:
oh yeah we really need to do this, Jeremy has an issue open for it https://github.com/deislabs/porter/issues/231